views/games.html

```diff
@@ -59,6 +65,12 @@ <h1>{{title}}</h1>
             <td class="col-xs-2">
                 {{players[game.blue_d]}}
             </td>
+            <td class="col-xs-2" data-game-length="{{game.length}}">
+                {{game.red_shots | length}}
+            </td>
+            <td class="col-xs-2" data-game-length="{{game.length}}">
```

Comment:
Bootstrap 3 uses a 12-column layout grid system. Each of these `col-xs-2` classes means that the table column/cell spans 2 columns of the layout grid columns. In order to add these two new table columns, we need to readjust the existing column classes, so that they all total 12 again. As of right now, we have now accounted for 16 of the 12 columns.

Again, just a tip. I'll handle this one.

Reply:
ahhhh I wish I was as full stack as you 😉 thanks mang